# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintData.h
# Commit: f946921b9173
# Full Hash: f946921b9173529d6ceec8200469747446c98a6a
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p3 - Move nsPrintData::mShrinkRatio to nsPrintJob::mShrinkToFitFactor. r=emilio
#   
#   Depends on D146974
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146975
# ==============================================================================

diff -r 5cd20e839a73 -r f946921b9173 layout/printing/nsPrintData.h
--- a/layout/printing/nsPrintData.h	Sun May 22 09:37:56 2022 +0000
+++ b/layout/printing/nsPrintData.h	Sun May 22 09:37:56 2022 +0000
@@ -64,7 +64,6 @@
   bool mIsAborted;          // tells us the document is being aborted
   bool mPreparingForPrint;  // see comments above
   int32_t mNumPrintablePages;
-  float mShrinkRatio;
 
  private:
   nsPrintData() = delete;